protocol.md

```diff
@@ -1078,13 +1078,24 @@ interface Hover {
 	contents: MarkedString | MarkedString[];
 	
 	/**
-	 * An optional range
+	 * An optional range is a range inside a text document 
+	 * that is used to visualize a hover, e.g. by changing the background color.
 	 */
 	range?: Range;
 }
 ```
 Where `MarkedString` is defined as follows:
 ```typescript
+/**
+ * The marked string is rendered:
+ * - as markdown if it is represented as a string
```

Comment:
Actually the idea was to use markdown :-) As the content definition says it is a `MarkedString | MarkedString[]`. So I will merge the PR in to make this more clear since it is todays state. I opened https://github.com/Microsoft/language-server-protocol/issues/94 to continue the discussion whether this was a good idea or not.